supernode/daemon/mgr/gc/gc_peer.go

```diff
@@ -40,8 +41,16 @@ func (gcm *Manager) gcPeers(ctx context.Context) {
 			continue
 		}
 
+		if peerState.ServiceDownTime == 0 {
+			cIDs, _ := gcm.dfgetTaskMgr.GetCIDAndTaskIDsByPeerID(ctx, peerID)
+			//if related task is not expired
+			if len(cIDs) > 0 {
+				continue
+			}
+		}
+
 		if peerState.ServiceDownTime != 0 &&
-			timeutils.GetCurrentTimeMillis()-peerState.ServiceDownTime < int64(gcm.cfg.PeerGCDelay) {
+			timeutils.GetCurrentTimeMillis()-peerState.ServiceDownTime < int64(gcm.cfg.PeerGCDelay/time.Millisecond) {
```

Comment:
@liudechi7 Could we move this PR ahead? And I think we can split this PR into two parts:
1. timestamp conversion error
2. If the peer exits abnormally without reporting status, the status of the peer will always exist.

And I think you can finish the first part in this PR firstly and complete the second task in the subsequent PR. WDYT?